docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    volumes:
+      - ".:/app"
+    command:
+      sh -c "./wait_for_db &&

```

Comment:
wait_for_db must be django_management_command
https://docs.djangoproject.com/en/4.0/howto/custom-management-commands/
https://www.mlr2d.org/modules/djangorestapi/09_command_to_wait_for_db